lib/galaxy/model/__init__.py

```diff
@@ -7880,6 +7883,9 @@ class Workflow(Base, Dictifiable, RepresentById):
     creator_metadata: Mapped[Optional[List[Dict[str, Any]]]] = mapped_column(JSONType)
     license: Mapped[Optional[str]] = mapped_column(TEXT)
     source_metadata: Mapped[Optional[Dict[str, str]]] = mapped_column(JSONType)
+    readme: Mapped[Optional[str]] = mapped_column(TEXT)
```

Comment:
Maybe `Text` instead of `TEXT`? I know we mostly use the latter, but, I suspect, there's no reason for that. Whereas the camelcase types are the more flexible, database-agnostic types and are recommended in the SA docs  ([ref](https://docs.sqlalchemy.org/en/20/core/type_basics.html#the-camelcase-datatypes)), whereas the uppercase are the exact type.

Reply:
Gotcha. Done.